guide-us-english.tex

```diff
@@ -431,7 +431,7 @@ \section{Some notes on Organizational Design}
 
 \textbf{Customer Relations, Legal / Compliance, and People Operations} are included here since they are necessary parts of organizations and will exist as independent Scrum Teams on their own, which all of the others may rely upon.
 
-A final note on the representation of the EAT \& EMS: in this diagram, they are shown as overlapping since some members sit on both of the teams. In very small organizations or implementations, the EAT \& EMS may consist entirely of the same team members.
+A final note on the representation of the EAT and EMT: in this diagram, they are shown as overlapping since some members sit on both of the teams. In very small organizations or implementations, the EAT and EMT may consist entirely of the same team members.
 
 \section{End Note}
```

Comment:
Like with the other commit, rather put all the MetaScrum stuff in a separate pull request. It needs a good look at, including discussion with His Jeffness.